src/navigator-card-body.jsx

```diff
@@ -310,9 +348,19 @@ export const NavigatorCardBody = ({
         }
     };
 
-    const handleContextMenu = () => {
-        const sel = path ? path[path.length - 1] : undefined;
-        setSelected([{ name: sel }]);
+    const handleContextMenu = (event) => {
+        let sel = getFilenameForEvent(event);
+        if (sel !== null && selected.length > 1) {
+            return;
+        }
+
+        if (sel === null) {
+            sel = path ? path[path.length - 1] : undefined;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-249-20240305-140311-42a1f07c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)